v3/src/components/case-table/use-rows.ts

```diff
@@ -44,17 +50,19 @@
         cells.forEach(cell => {
           const colIndex = Number(cell.getAttribute("aria-colindex")) - 2
           const attr = data?.attributes[colIndex]
+          const cellSpan = cell.querySelector(".cell-span")
           if (data && caseId && attr) {
             const strValue = data.getValue(caseId, attr.id)
             const numValue = data.getNumeric(caseId, attr.id)
             const formatStr = attr.format || kDefaultFormatStr
-            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue
-            cell.textContent = formatted
+            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue;
+            (cellSpan || cell).textContent = formatted
```

Comment:
No need for "cell" here
```suggestion
            cellSpan.textContent = formatted
```